vpr/src/route/rr_graph.cpp

```diff
@@ -2937,8 +2937,8 @@ static RRNodeId pick_best_direct_connect_target_rr_node(const t_rr_graph_storage
 
         for (RRNodeId to_rr : candidate_rr_nodes) {
             VTR_ASSERT(rr_graph.node_type(to_rr) == IPIN);
-            float to_dist = std::abs(rr_nodes.node_xlow(from_rr) - rr_nodes.node_xlow(to_rr))
-                            + std::abs(rr_nodes.node_ylow(from_rr) - rr_nodes.node_ylow(to_rr));
+            float to_dist = std::abs(rr_graph.node_xlow(from_rr) - rr_graph.node_xlow(to_rr))
```

Comment:
Consider to use the newly added API ``node_length()``

Reply:
I am not sure if `node_length()` applies here since this is dealing with two distinct nodes (`from_rr`, `to_rr`) rather than one. Am I missing something?

Reply:
@ethanroj23 My bad. The node_length() is not applicable here. Your changes are good.